Activity Log: Add missing icons

momo-ozawa opened the pull request:
Closes #20597 

## Description
Adds mapping for videopress icon on the activity log

## How to test
1. Switch to a site that has activity log enabled
2. Upload a video via videopress
3. Go to the activity log
4. ✅ The video upload event has an icon

Before | After
-- | --
<img src="https://user-images.githubusercontent.com/6711616/234225734-f46b4e05-9679-4bb4-b05f-ef684a45e659.png" width=200> | <img src="https://user-images.githubusercontent.com/6711616/234225136-f29302d2-26ca-4d53-98b0-85584a24897a.png" width=200> 

## Regression Notes
1. Potential unintended areas of impact
n/a

5. What I did to test those areas of impact (or what existing automated tests I relied on)
n/a

6. What automated tests I added (or what prevented me from doing so)
n/a


PR submission checklist:

- [x] I have completed the Regression Notes.
- [x] I have considered adding unit tests for my changes.
- [x] I have considered adding accessibility improvements for my changes.
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

UI Changes testing checklist:
- [x] Portrait and landscape orientations.
- [x] Light and dark modes.
- [x] Fonts: Larger, smaller and bold text.
- [x] High contrast.
- [x] VoiceOver.
- [x] Languages with large words or with letters/accents not frequently used in English.
- [x] Right-to-left languages. (Even if translation isn’t complete, formatting should still respect the right-to-left layout)
- [x] iPhone and iPad. 
- [x] Multi-tasking: Split view and Slide over. (iPad)

Commit:
Add video icon mapping for activity log

hassaanelgarem reviewed (commented):
I searched through the wpcom codebase (fbhepr%2Skers%2Sjcpbz%2Sjc%2Qpbagrag%2Syvo%2Snpgvivgl%2Qybt%2Qgb%2Qj3%2Qnpgvivgl%2Qfgernz%2Qvgrz%2Snpgvivgvrf%2S-og) and I found some more icons that need mapping:
- `status`
    - Used by `Activity_Stream_Item`, the base activity class
    - It appears that all other subclasses override the icon value, but it could happen that a class doesn't.
- `cart`
    - Used by `Activity_Stream_Item_Shop_Order`
    - It's probably only used by Woo but I'm not sure
- `custom-post-type`
    - Used by `Activity_Stream_Item_Feedback_Published`
- `multiple-users`
    - Used by `Activity_Stream_Item_Sftp_User`
- `audio`
    - Used by `Activity_Stream_Item_Attachment`

Commit:
Add missing activity log icons

# Conflicts:
#	WordPress/Classes/ViewRelated/Activity/WPStyleGuide+Activity.swift

momo-ozawa commented:
Thanks @hassaanelgarem! Fixed in 33c4eda
